app/helpers.rb

```diff
@@ -0,0 +1,12 @@
+class Helpers
+#helpers
+  def self.current_user(user)
```

Comment:
Using the variable `user` to refer to a `session` is a little confusing. The session can store different data than the `User` model. Also, the session should be directly available in the helper methods, without having to pass it as a parameter.